### ISSUE TITLE
Validating a specific type causes compilation to fail

### ISSUE THREAD

**Issue by jff1625** [samchon/typia#1431]
- **Typia Version**: 7.3.2
  - **Expected behavior**: tsc compiles successfully
  - **Actual behavior**: error TS2367: This comparison appears to be unintentional because the types 'string' and 'any[]' have no overlap.

Some kind of incompatibility with a 3rd-party type lib `@types/google-publisher-tag`
If I try to validate their type `googletag.GeneralSize` the compilation fails, with the above error.

See attached minimal reproduction zip. `bun run build` or simply `tsc` will trigger the error.

Please advise if the problem comes from you or from google.

### 💻 Code occuring the bug
```typescript
import { is } from "typia";
is<googletag.GeneralSize>(1);
```
[typia-GeneralSize.zip](https://github.com/user-attachments/files/18128762/typia-GeneralSize.zip)

**Comment by jboyesreach** [comment]
was this fixed? I can't replicate it today.

**Comment by samchon** [comment]
Will you give me a reproducible repo?

**Comment by jff1625** [comment]
No, I can no longer replicate it. It's very strange. All the version numbers in the package.json's in node_modules/ are the same.
Anyway, please close the issue. Thanks for your time.44
